.github/workflows/main.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
         java_version: [8, 11, 17, 21]
         include:
           - java_version: '8'
-            included_modules: '-pl !code-execution-engines/langchain4j-code-execution-engine-graalvm-polyglot,!langchain4j-cassandra,!langchain4j-infinispan,!langchain4j-neo4j,!langchain4j-opensearch'
+            included_modules: '-pl !code-execution-engines/langchain4j-code-execution-engine-graalvm-polyglot,!langchain4j-cassandra,!langchain4j-infinispan,!langchain4j-neo4j,!langchain4j-opensearch,!langchain4j-azure-ai-search'
```

Comment:
No, this needs to use Mockito 5 as  `com.azure.search.documents.models.SearchResult` is a final class. Mockito 4 cannot modify it:

```
org.mockito.exceptions.base.MockitoException: 
Cannot mock/spy class com.azure.search.documents.models.SearchResult
Mockito cannot mock/spy because :
 - final class

	at dev.langchain4j.store.embedding.azure.search.AzureAiSearchContentRetrieverTest.testFromAzureScoreToRelevanceScore_SIMILARITY(AzureAiSearchContentRetrieverTest.java:15)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)

Reply:
Marking this as "resolved" as I don't think we can do anything more.